pom.xml

```diff
@@ -55,6 +55,8 @@
         <azure.media.version>0.9.8</azure.media.version>
         <azure.client.authentication.version>1.6.1</azure.client.authentication.version>
         <azure.cosmosdb.version>2.2.0</azure.cosmosdb.version>
+        <azuremonitor.micrometer.registry.version>1.1.2</azuremonitor.micrometer.registry.version>
+        <micrometer.core.version>1.1.2</micrometer.core.version>
```

Comment:
This only defines the version properties, could you also add the `<dependency>.....</dependency>`?